velox/common/encode/Base64.h

```diff
@@ -41,119 +40,97 @@ class Base64 {
   /// within the encoding base.
   using ReverseIndex = std::array<uint8_t, kReverseIndexSize>;
 
-  /// Padding character used in encoding.
-  static const char kPadding = '=';
-
-  /// Encodes the specified number of characters from the 'data'.
-  static std::string encode(const char* data, size_t len);
+  /// Encodes the specified number of characters from the 'input'.
+  static std::string encode(std::string_view input, size_t len);
 
   /// Encodes the specified text.
-  static std::string encode(folly::StringPiece text);
+  static std::string encode(std::string_view text);
```

Comment:
`folly::StringPiece` is equivalent to `std::string`

Reply:
https://github.com/facebookincubator/velox/pull/10371#discussion_r1711236265
I've made the necessary changes based on the comment above.

Reply:
I see. But there is already an API with std::string_view above.